packages/schematics/angular/web-worker/schema.d.ts

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
This file should not be needed.

`schema.d.ts` are generated from `schema.json` during build time using quicktype.

Reply:
Removed